Assets/MRTK.ThirdParty/MRTK-Quest/Editor/MRTKQuestConfigurationChecker.cs

```diff
@@ -0,0 +1,203 @@
+            string cscFilePath = Path.Combine(Application.dataPath, "csc.rsp");
+
+            // Each line of the csc file
+            List<string> cscFileLines = new List<string>();
+
+            // List of the warning numbers after "-nowarn: " in the csc file
+            List<string> warningNumbers = new List<string>();
+
+            // List of new warning numbers to add to the csc file
+            List<string> warningNumbersToAdd = new List<string>()
+            {
+                "618",
+                "649"
+            };
+
+            using (StreamReader streamReader = new StreamReader(cscFilePath))
+            {
+                while (streamReader.Peek() > -1)
+                {
+                    string cscFileLine = streamReader.ReadLine();
+
+                    if (cscFileLine.Contains("-nowarn"))
+                    {
+                        string[] currentWarningNumbers = cscFileLine.Split(',', ':');
+                        warningNumbers = currentWarningNumbers.ToList();
+
+                        // Remove "nowarn" from the warningNumbers list
+                        warningNumbers.Remove("-nowarn");
+
+                        foreach (string warningNumberToAdd in warningNumbersToAdd)
+                        {
+                            // Add the new warning numbers if they are not already in the file
+                            if (!warningNumbers.Contains(warningNumberToAdd))
+                            {
+                                warningNumbers.Add(warningNumberToAdd);
+                            }
+                        }
+
+                        cscFileLines.Add(string.Join(",", warningNumbers));
+                    }
+                    else
+                    {
+                        cscFileLines.Add(cscFileLine);
+                    }
+                }
+            }
+
+            using (StreamWriter streamWriter = new StreamWriter(cscFilePath))
+            {
+                foreach (string cscLine in cscFileLines)
+                {
+                    if (cscLine.StartsWith("1701"))
+                    {
+                        string warningNumbersJoined = string.Join(",", warningNumbers);
+                        streamWriter.WriteLine(string.Concat("-nowarn:", warningNumbersJoined));
+                    }
+                    else
+                    {
+                        streamWriter.WriteLine(cscLine);
+                    }
+                }
+            }
+
+            //Debug.Log($"Saving {cscFilePath}");
+        }
+
+
+        /// <summary>
+        /// Locates the files that match the specified name within the Assets folder structure.
+        /// </summary>
+        /// <param name="fileName">The name of the file to locate (ex: "TestFile.asmdef")</param>
+        /// <returns>Array of FileInfo objects representing the located file(s).</returns>
+        public static FileInfo[] FindFilesInAssets(string fileName)
```

Comment:
Can we use FileUtilites.FindFilesInAssets in this case, or is there a key functionality difference that's important to preserve?

Reply:
Its mostly due to namespace/dependency concerns/not wanting to make changes to MRTK-Quest that aren't reflected in their base repository as well. @provencher is it ok if we make the alterations needed here to reflect this being a subset of mainline MRTK?

Reply:
Sure thing!